lib/libutee/include/pta_imx_digprog.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2021 NXP
+ */
+#ifndef __PTA_IMX_DIGPROG_H__
+#define __PTA_IMX_DIGPROG_H__
+
+#define PTA_DIGPROG_UUID \
+	{ \
+		0xbfdbe409, 0x3302, 0x4abf, \
+		{ \
+			0x99, 0xbc, 0x5f, 0x62, 0x32, 0x26, 0xa7, 0x39 \
+		} \
+	}
```

Comment:
Please format like the other `pta_*.h` files, and same for OCOTP

Reply:
Ok